common/app/model/PressedPage.scala

```diff
@@ -134,8 +134,12 @@ case class PressedPage(
       item <- pressedCollection.curatedPlusBackfillDeduplicated
       id <- {
         item match {
-          case curatedContent: CuratedContent =>
-            curatedContent.content.sectionId ++ curatedContent.content.tags.map(_.id)
+          case curatedContent: pressed.CuratedContent => {
+            curatedContent.properties.maybeContent match {
+              case Some(content) => content.metadata.sectionId.map(_.value) ++ content.tags.tags.map(_.id)
+              case None          => Nil
+            }
+          }
```

Comment:
I think an alternative would be to map over maybeContent and then use `getOrElse(nil)` after the map. This would yield the same result but it's a personal preference as to which is easier to read so no blocker here

Reply:
eg
```
 curatedContent.properties.maybeContent
      .map(content => content.metadata.sectionId.map(_.value) ++ content.tags.tags.map(_.id))
      .getOrElse(Nil)
 ```